src/app/backend-api.service.ts

```diff
@@ -69,8 +74,15 @@ export class BackendRoutes {
   static RoutePathGetNextNFTShowcase = "/api/v0/get-next-nft-showcase";
   static RoutePathGetNFTCollectionSummary = "/api/v0/get-nft-collection-summary";
   static RoutePathGetNFTEntriesForPostHash = "/api/v0/get-nft-entries-for-nft-post";
+
+  // Verify
```

Comment:
Nit: since "Verify" is already used, we should either move this endpoint under the other header or change the title of this section